src/apps/competitions/tasks.py

```diff
@@ -680,3 +680,11 @@ def create_competition_dump(competition_pk, keys_instead_of_files=True):
         logger.info(f"Finished creating competition dump: {temp_comp_dump.pk} for competition: {comp.pk}")
     except ObjectDoesNotExist:
         logger.info("Could not find competition with pk {} to create a competition dump".format(competition_pk))
+
+
+@app.task(queue='site-worker', soft_time_limit=60 * 5)
+def do_phase_migrations():
+    competitions = Competition.objects.filter(is_migrating=False)
```

Comment:
I'm not 100% sure this will work as intended, but consider:
`competitions = Competition.objects.filter(is_migrating=False, phases__auto_migration=True)`
This should be tested, but I _think_ this should filter out any competitions that don't have any phases with `auto_migration=True`, will save some work for the site_worker if that is true. If this doesn't work, I'm pretty damn sure we could come up with a query that _will_ do that.

Reply:
Added this logic and this cut down on db calls. I didn't know about the `<model>__<model_field>` so that was really helpful. Thanks!